src/simplewallet/simplewallet.cpp

```diff
@@ -553,6 +559,14 @@ namespace
     }
     return true;
   }
+
+  void print_secret_key(const crypto::secret_key &k)
+  {
+    const epee::wipeable_string s = string_tools::pod_to_hex(k);
```

Comment:
Doesn't `pod_to_hex` create a temporary `std::string` whose data might stay in the memory?